code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -499,7 +499,9 @@ generate/load female uniform sprites matching all previously decided variables
 	if(override_state)
 		t_state = override_state
 	else
-		if(isinhands && item_state)
+		if (!isinhands && mob_overlay_state)
+			t_state = mob_overlay_state
+		else if(item_state)

```

Comment:
Uh, surely there exists items that have item_state defined but not a corresponding mob icon ?

Reply:
Yeah, but that's a bug on their end. This was how this proc was expected to behave for years, I accidentally broke it without realizing it.

Reply:
Alright, i'm going to trust you checked it's nothing common.